apps/ui/src/helpers/connectors/coinbase.ts

```diff
@@ -19,23 +19,7 @@ export default class Walletlink extends Connector {
     }
   }
 
-  removeHashFromLocalStorage() {
-    if (!localStorage) return;
-
-    const keys: string[] = [];
-
-    for (let i = 0; i < localStorage.length; i++) {
-      const key = localStorage.key(i) as string;
-
-      if (key.startsWith('-walletlink:')) {
-        keys.push(key);
-      }
-    }
-
-    keys.forEach(key => localStorage.removeItem(key));
```

Comment:
On logout, now It doesn't remove these localstorage variables, maybe it was meant to do something? maybe its fine with newer versions of coinbase, we can try without this
<img width="149" alt="Untitled 4" src="https://github.com/user-attachments/assets/ddd7f06b-11d0-407e-af36-bae6ca294804" />

Reply:
The screenshot is when I tried last night, I've cleared existing keys before connecting my wallet. I am using coinbase chrome extension v3.105.0

Reply:
Will try again with the extension